worker/s3.go

```diff
@@ -105,6 +106,7 @@ func putObject(service *s3.S3, objectName string, objectContent io.ReadSeeker, b
 		Body:   objectContent,
 	}, func(d *s3manager.Uploader) {
 		d.MaxUploadParts = 1
+		d.Concurrency = 20
```

Comment:
why do you set this to 20?

Reply:
This commit is contributed by @seong-yeop, I just made a little modification based on his commit.

Reply:
Another question, why does `MaxUploadParts` set to 1?